### PR TITLE
add configuration

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
https://github.com/open-orchestra/open-orchestra-base-api-bundle/pull/28
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/918
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/20
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/64
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/362/files
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/62/files
https://github.com/open-orchestra/open-orchestra-model-interface/pull/91
https://github.com/open-orchestra/open-orchestra-docs/pull/55

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+open-orchestra-libs
+===================
+
+| Service       | Badge         |
+| ------------- |:-------------:|
+| Travis | [![Build Status](https://travis-ci.org/open-orchestra/open-orchestra-libs.svg)](https://travis-ci.org/open-orchestra/open-orchestra-libs) |
+| CodeClimate (quality) | [![Code Climate](https://codeclimate.com/github/open-orchestra/open-orchestra-libs/badges/gpa.svg)](https://codeclimate.com/github/open-orchestra/open-orchestra-libs) |
+| CodeClimate (coverage) | [![Test Coverage](https://codeclimate.com/github/open-orchestra/open-orchestra-libs/badges/coverage.svg)](https://codeclimate.com/github/open-orchestra/open-orchestra-libs/coverage) |
+| Sensio Insight | [![SensioLabsInsight](https://insight.sensiolabs.com/projects/e7fb8f90-e8f0-4da4-9375-42e699038b79/big.png)](https://insight.sensiolabs.com/projects/e7fb8f90-e8f0-4da4-9375-42e699038b79) |
```
Comment (nicolasThal) [inline README.md]: this badge might not be the good one

File: README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+| VersionEye | [![Dependency Status](https://www.versioneye.com/user/projects/551e87a7971f781c480001aa/badge.svg?style=flat)](https://www.versioneye.com/user/projects/551e87a7971f781c480001aa) |
```
Comment (nicolasThal) [inline README.md]: same

Comment (nicolasThal) [inline README.md]: Use this one : [Dependency Status](https://www.versioneye.com/user/projects/55967d7d6166340021000002)

File: README.md (hunk 1 above)
Comment (nicolasThal) [inline README.md]: Use this one : [SensioLabsInsight](https://insight.sensiolabs.com/projects/d7707acf-621b-4c49-adcd-ad14f8f86bb2)

**Comment by nicolasThal** [comment]
Do not forgot to add the array with all the badges to the quality_summary page of the documentation

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**